ruby/project_advanced_building_blocks.md

```diff
@@ -220,7 +221,7 @@ There's also [an entry on Bubble Sort on Wikipedia](http://en.wikipedia.org/wiki
 * [Dom Goj's solution](https://github.com/booyakuhhsha/rubyAdvancedBuildingBlocks/blob/master/bubblesort.rb)
 * [grzegorzzajac1989's solution](https://github.com/grzegorzzajac1989/theOdinProject/blob/master/ruby/advenced_building_block/bubble_sort.rb)
 * [Jason Keeney's solution](https://github.com/jkeeney/Advanced-Building-Blocks/blob/master/bubble_sort.rb)
-* [Hassan's solution](https://github.com/HassanTC/Ruby_TheOdinProject/blob/master/The%20odin%20project%20(Ruby%20projects/Basic%20Ruby/bubble_sort.rb)
+* [Hassan's solution](https://github.com/HassanTC/Ruby_TheOdinProject/blob/master/The%20odin%20project%20(Ruby%20projects/Basic%20Ruby/bubble_sort.rb))
```

Comment:
Hi @brunoparga ,
This placement of this closing `)` is not correct. Can you please change the link address to:
`https://github.com/HassanTC/Simple_web_projects/blob/master/Ruby_TheOdinProject/The%20odin%20project%20(Ruby%20projects)/Basic%20Ruby/bubble_sort.rb`

Reply:
Done.